docs/developers/tutorials/staking-contract.md

```diff
@@ -247,22 +241,16 @@ You have to press "space" between the words, not "enter"!
 
 [comment]: # (mx-context-auto)
 
-### Deploying the contract
+### Deploying the contract on devnet
```

Comment:
Above, `mxpy wallet derive...` should be re-written as:

```
mkdir -p ~/MyTestWallets
mxpy wallet convert --in-format=raw-mnemonic --out-format=pem --outfile=~/MyTestWallets/tutorialKey.pem

```

Reply:
done